dsa-solutions/lc-solutions/0000-0099/0034-Find-first-and-last-position-in-sorted-array.md

```diff
@@ -0,0 +1,170 @@
+---
+id: Find First and Last Position of Element in Sorted Array
```

Comment:
replace `id: Find First and Last Position of Element in Sorted Array` to `id: find-first-and-last-position-of-element-in-sorted-array`

Reply:
@PradnyaGaitonde Complete your task

Reply:
@Ajay-Dhangar Completed it.